docs/prerequisites.adoc

```diff
@@ -50,16 +38,18 @@ If you are using Oracle Linux, you can install Terraform as follows:
 yum -y install oraclelinux-developer-release-el7 && yum -y install terraform
 ----
 
-==== Installing Terraform on macOS
+=== Installing Terraform on macOS
 
 If you are using a Mac, you can install Terraform as follows:
```

Comment:
I would reword this to "If you are using macOS, you can use Homebrew to install Terraform"
 with a link to https://brew.sh/. Keep in mind that Homebrew isn't installed by default on macOS, so you need to include that as a step.